src/debug/ee/amd64/doc/createOpcodes.cpp

```diff
@@ -0,0 +1,253 @@
+#include <stdio.h>
```

Comment:
'doc' isn't the sub-directory I'd expect to look in to find these tools. How about 'opcode_gen' or some other name that relates to tables, generation, or disassembly?